src/metabase/lib/inflate.cljc

```diff
@@ -0,0 +1,7 @@
+(ns metabase.lib.inflate
```

Comment:
Do inflate/deflate need to be different namespaces if they only contain one defmulti each?

Reply:
Nope, I need to go consolidate this stuff at some point. This is still in a very rough draft PoC state and I haven't cleaned up everything yet